src/EditorFeatures/Core/Implementation/TodoComment/AbstractTodoCommentIncrementalAnalyzer.cs

```diff
@@ -91,9 +87,31 @@ public Task DocumentResetAsync(Document document, CancellationToken cancellation
             }
         }
 
+        private async Task<IList<TodoComment>> GetTodoCommentsAsync(Document document, ImmutableArray<TodoCommentDescriptor> tokens, CancellationToken cancellationToken)
+        {
+            var client = await _workspace.GetRemoteHostClientAsync(cancellationToken).ConfigureAwait(false);
+            if (client != null && !document.IsOpen())
```

Comment:
since todo is syntax analyzer, doing whole as project is actually worse. but I will open a bug for designer scanner.